data/Nanthakumaran-S.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Nanthakumaran S",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Software Developer | **Cloud** & **DevOps** Enthusiast | Open Source Beginner",
+  "tags": ["Open Source", "DevOps", "Kubernetes", "Docker", "Node.js", "React.js"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/nanthakumaran_" },
+    { "icon": "FaGithub", "url": "https://github.com/nanthakumaran-s" },
+    { "icon": "FaLinkedin", "url": "https://linkedin.com/in/nanthakumaran-s" },
+    { "icon": "website", "url": "https://nanthakumaran.com" }
```

Comment:
This is not a valid icon, please select from `Fa` or `Si` icon set, you can search the here https://linkfree.eddiehub.io/icons

Btw I see many people using `website` is this somewhere in the docs that we forgot to update?

Reply:
Sure will change them... I have found similar usage in others' JSON files.